apps/communications/contacts/test/marionette/import_sdcard_test.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+var Contacts = require('./lib/contacts');
+var assert = require('assert');
+
+marionette('Import from SDCARD', function() {
+  var profile = Contacts.config;
+  // Enable device storage
+  var prefs = ['device.storage.enabled',
+        'device.storage.testing',
+        'device.storage.prompt.testing'];
+  prefs.forEach(pref => {
+    profile[pref] = true;
+  });
+  var client = marionette.client({
+    profile: profile,
+    desiredCapabilities: { raisesAccessibilityExceptions: false }
+  });
+  var actions, subject, selectors;
+
+  setup(function() {
+    // Remove all files in temp device storage.
+    client.fileManager.removeAllFiles();
+
+    // Add the default vcard file
+    client.fileManager.add({
+      type: 'text',
+      filePath: 'apps/communications/contacts/test/marionette/data/vcard_4.vcf'
+    });
+
+    subject = new Contacts(client);
+    actions = client.loader.getActions();
+    subject.actions = actions;
```

Comment:
Injecting this dependency doesn't look correct, as the `subject` is already constructed with `client`. 

I'd retrieve the action at [this line](https://github.com/mozilla-b2g/gaia/blob/master/apps/communications/contacts/test/marionette/lib/contacts.js#L287) (which is the only one using an action) with directly with:
`var action = this.client.loader.getActions();`